lib/Dialect/ESI/runtime/cosim/cosim_dpi_server/DpiEntryPoints.cpp

```diff
@@ -299,47 +332,50 @@ DPI int sv2cCosimserverMMIORegister() {
 }
 
 DPI int sv2cCosimserverMMIOReadTryGet(uint32_t *address) {
-  assert(server);
-  LowLevel *ll = server->getLowLevel();
-  std::optional<int> reqAddress = ll->readReqs.pop();
-  if (!reqAddress.has_value())
-    return -1;
-  *address = reqAddress.value();
-  ll->readsOutstanding++;
-  return 0;
+  // assert(server);
+  // LowLevel *ll = server->getLowLevel();
+  // std::optional<int> reqAddress = ll->readReqs.pop();
+  // if (!reqAddress.has_value())
+  return -1;
+  // *address = reqAddress.value();
```

Comment:
Calling this function is not actually a no-no since I didn't want to break compatibility. Plus, this function is called every cycle, so... Writing a response would be, however, so that's the right place for a message and there's an assert there.